pitest-maven-verification/src/test/java/org/pitest/PitMojoIT.java

```diff
@@ -398,32 +323,73 @@ private File buildFile(File base, String... pathParts) {
 	  StringBuilder path = new StringBuilder(base.getAbsolutePath());
 
 	  for (String part : pathParts) {
-		  path.append(File.separator).append(part);
+	      if(!StringUtils.isBlank(part)) {
```

Comment:
Given that this is a private helper function within test code, I think its not necessary to program that defensively.  I removed the check for isBlank and added javadoc.  I can easily throw an IllegalArgumentException if pathParts is empty or any item within pathParts is empty if you think that is warranted.